HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,562 @@
+            "PropertyRequirements": {
+                "AuthorizationScope": {},
+                "EntitlementId": {},
+                "ExpirationDate": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "InstallDate": {},
+                "LicenseInfoURI": {
+                    "ReadRequirement": "Recommended"
+                },
+                "LicenseOrigin": {},
+                "LicenseType": {},
+                "Manufacturer": {},
+                "Removable": {},
+                "SKU": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Status": {
+                    "PropertyRequirements": {
+                        "State": {
+                            "ReadRequirement": "Recommended"
+                        }
+                    }
+                }
+            }
+        },
+        "LicenseService": {
+            "ReadRequirement": "Recommended",
+            "Purpose": "For products that utilize software licensing, supporting the installation and updating of licenses through the Redfish interface is highly recommended.",
+            "PropertyRequirements": {
+                "Licenses": {}
+            },
+            "ActionRequirements": {
+                "Install": {
+                    "ReadRequirement": "Recommended",
+                    "Purpose": "Encourage common toolchain usage for mass operations.  While individual licenses will be highly dependent on vendor definitions, the process to install those licenses should be vendor-independent.",
+                    "Parameters": {
+                        "LicenseFileURI": {}
+                    }
+                }
+            }
+        },
+        "LogEntry": {
+            "PropertyRequirements": {
+                "Created": {},
+                "EntryType": {},
+                "EventGroupId": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Message": {
+                    "ReadRequirement": "Recommended"
+                },
+                "MessageId": {},
+                "MesageArgs": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Severity": {}
+            }
+        },
+        "LogService": {
+            "PropertyRequirements": {
+                "DateTime": {},
+                "DateTimeLocalOffset": {},
+                "Entries": {},
+                "LogEntryType": {},
+                "LogPurposes": {
+                    "ReadRequirement": "Recommended"
+                },
+                "ServiceEnabled": {
+                    "ReadRequirement": "Recommended"
+                }
+            },
+            "ActionRequirements": {
+                "ClearLog": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "Manager": {
+            "PropertyRequirements": {
```

Comment:
DateTime and DateTimeOffset? Also write requirements for those?

Reply:
I made both as recommended R/W in case the time is supplied by another manager?

Reply:
That's probably okay. Probably not typical, but we shouldn't be too restrictive for those cases...